SPEC.md

```diff
@@ -2,33 +2,37 @@
+The specification defines an image format, image discovery mechanism, and execution environment that can exist in several independent implementations.
+The core goals of the specification include:
 
-* Design for fast downloads and starts of the containers
-* Ensure images are cryptographically verifiable and highly-cacheable
-* Design for composability and independent implementations
-* Use common technologies for crypto, archive, compression and transport
-* Use the DNS namespace to name and discover container images
+* Designing for fast downloads and starts of App Containers
+* Ensuring images are cryptographically verifiable and highly cacheable
+* Designing for composability and independent implementations
+* Using common technologies for cryptography, archiving, compression and transport
+* Using the DNS namespace to name and discover images
 
-To achieve these goals this specification is split out into a number of smaller sections.
+The specification consists of several key sections:
 
 1. The **[App Container Image](#app-container-image)** defines: how files are assembled together into a single image, verified on download and placed onto disk to be run.
 
-2. The **[App Container Executor](#app-container-executor)** defines: how an app container image on disk is run and the environment it is run inside including cgroups, namespaces and networking.
+2. The **[App Container Executor](#app-container-executor)** defines: how applications are grouped into execution units and the environment they are run inside (including, for example, filesystem layout, resource constraints, and networking).
 
-    * The [Metadata Server](#app-container-metadata-service) defines how a container can introspect and get a cryptographically verifiable identity from the execution environment.
+    * The [Pod Manifest](#pod-manifest) defines how one or more App Container Images are grouped into a deployable, executable unit.
 
-3. The **[App Container Image Discovery](#app-container-image-discovery)** defines: how to take a name like example.com/reduce-worker and translate that into a downloadable image.
+    * The [Metadata Service](#app-container-metadata-service) defines how an app can introspect and get a cryptographically verifiable identity from the execution environment.
+
+3. **[App Container Image Discovery](#app-container-image-discovery)** defines: how to take a name like `example.com/reduce-worker` and translate that into a downloadable image.
 
 
 ## Example Use Case
 
 To provide context to the specs outlined below we will walk through an example.
 
-A user wants to launch a container running three processes.
+A user wants to launch an "App Container" running three processes.
 The three processes the user wants to run are the apps named `example.com/reduce-worker-register`, `example.com/reduce-worker`, and `example.com/reduce-backup`.
 First, the executor will check the cache and find that it doesn't have images available for these apps.
```

Comment:
cache is not introduced but used in this line as if a necessary part of an executor. Perhaps just strike this line and start on line 35, "First, an executor will make an HTTPS request..."